readme.md

```diff
@@ -57,8 +57,7 @@ Install the Arkeo binary
 ```bash
 git clone [https://github.com/arkeonetwork/arkeo](https://github.com/arkeonetwork/arkeo)
 cd arkeo
-git checkout <version hash, found at
-https://github.com/arkeonetwork/infra/blob/master/arkeo-stack/mainnet.yaml#L3>
+git checkout ab05b124336ace257baa2cac07f7d1bfeed9ac02
```

Comment:
i would just set this to master, that is good enough and doesn't require constant updating

Reply:
I know a lot of chains work with pre-releases, non-essential releases and the final production versions.

Having either the git commit code or version tag (the latter is preferred since it is way easier) allows validators to control which version of the binary they are installing. That might also be required for when a validator syncs from block 0 to create an archive node.